turtlebot2_demo.repos

```diff
@@ -42,7 +42,7 @@ repositories:
   eProsima/Fast-RTPS:
     type: git
     url: https://github.com/eProsima/Fast-RTPS.git
-    version: 4ae016fb90ea383e00b956fcba977f486894a0e5
+    version: master
```

Comment:
if we use fastRTPS master here I think it would make sense to move the boost dependencies installation in the [turtlebot section of the dockerfile](https://github.com/ros2/ci/blob/master/linux_docker_resources/Dockerfile#L89). If they are needed for the turtlebot2_demo, otherwise they can be removed.